lib/jekyll/page_without_a_file.rb

```diff
@@ -2,8 +2,10 @@
 
 module Jekyll
   class PageWithoutAFile < Page
+    # rubocop:disable Naming/MemoizedInstanceVariableName
     def read_yaml(*)
```

Comment:
Rubocop wants to call this method `@read_yaml`

Reply:
The method is already called `read_yaml`, Rubocop wants us to name the `@data` instance `@read_yaml` no?

Reply:
Oh! Yeah, ok, that makes a _lot_ more sense 🤦‍♂️